pkgs/development/tools/parse/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl }:
+
+let
+  version = "3.0.0";
+
+in stdenv.mkDerivation rec {
+
+  name = "parse-${version}";
```

Comment:
We should call it `parse-cli-bin-${version}` as it's in binary form and real name of the software is `parse-cli`